tests/test_id_io.py

```
@@ -0,0 +1,1207 @@
+            "protein_accessions": np.array([], dtype=object),
+            "psm_metavalues": np.array([], dtype=object),
+            "spectrum_metavalues": np.array([], dtype=object),
+        })
+        psms_df = pd.DataFrame([row])
+
+        out = str(tmp_path / "out.idparquet")
+        save_psms_from_scratch(out, psms_df)
+
+        psms2, *_ = load_dataframes(out)
+        assert len(psms2) == 1
+        assert psms2["is_decoy"].iloc[0] is False or psms2["is_decoy"].iloc[0] == False, (
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `False`; use `not psms2["is_decoy"].iloc[0]:` for false checks

Replace with `not psms2["is_decoy"].iloc[0]`

(E712)